### PR TITLE
unvanquished: 0.52.0 -> 0.52.1

### PR DESCRIPTION
**Pull request by necessarily-equal**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested execution **of the client only** (the server shouldn't have changed)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Tests fail https://logs.nix.ci/?key=nixos/nixpkgs.127771&attempt_id=5834de57-4e49-4501-af17-f65b6003b827

**Comment by necessarily-equal** [comment]
should be fixed now 
https://logs.nix.ci/?attempt_id=49784fb7-68c0-42b9-8af7-b29df812d7b9&key=nixos%2Fnixpkgs.127771

File: pkgs/games/unvanquished/default.nix [hunk 1]
```diff
@@ -77,15 +77,14 @@ let
     chmod +x $out/bin/${wrappername}
   '';
 
-
   unvanquished-assets = stdenv.mkDerivation {
     pname = "unvanquished-assets";
     inherit version src;
 
-    outputHash = "sha256:1fy85cjnjk9rrqkhgx5701inff2yv14hnxglzx3209c553gn31n7";
+    outputHash = "sha256:084jdisb48xyk9agjifn0nlnsdnjgg32si8zd1khsywd0kffplzx";
```
Comment (SuperSandro2000) [inline pkgs/games/unvanquished/default.nix:84]:
```suggestion
    outputHash = "084jdisb48xyk9agjifn0nlnsdnjgg32si8zd1khsywd0kffplzx";
```

File: pkgs/games/unvanquished/default.nix [hunk 2]
```diff
@@ -77,15 +77,14 @@ let
     chmod +x $out/bin/${wrappername}
   '';
 
-
   unvanquished-assets = stdenv.mkDerivation {
     pname = "unvanquished-assets";
     inherit version src;
 
-    outputHash = "sha256:1fy85cjnjk9rrqkhgx5701inff2yv14hnxglzx3209c553gn31n7";
+    outputHash = "084jdisb48xyk9agjifn0nlnsdnjgg32si8zd1khsywd0kffplzx";
```
Comment (SuperSandro2000) [inline pkgs/games/unvanquished/default.nix:84]:
```suggestion
    outputHash = "sha265:084jdisb48xyk9agjifn0nlnsdnjgg32si8zd1khsywd0kffplzx";
```

File: pkgs/games/unvanquished/default.nix [hunk 3]
```diff
@@ -77,15 +77,14 @@ let
     chmod +x $out/bin/${wrappername}
   '';
 
-
   unvanquished-assets = stdenv.mkDerivation {
     pname = "unvanquished-assets";
     inherit version src;
 
-    outputHash = "sha256:1fy85cjnjk9rrqkhgx5701inff2yv14hnxglzx3209c553gn31n7";
+    outputHash = "sha265:084jdisb48xyk9agjifn0nlnsdnjgg32si8zd1khsywd0kffplzx";
```
Comment (SuperSandro2000) [inline pkgs/games/unvanquished/default.nix:84]:
```suggestion
    outputHash = "sha256:084jdisb48xyk9agjifn0nlnsdnjgg32si8zd1khsywd0kffplzx";
```

**Pull request merged**